go.sum

```diff
@@ -131,6 +131,8 @@ github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa/go.mod h1:z
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
 github.com/codeready-toolchain/api v0.0.0-20210412143528-2a936530ad35 h1:wLjMKdIpKjBHcpQ/M9abx5ExuWac+aNwPGLGKugZyAg=
 github.com/codeready-toolchain/api v0.0.0-20210412143528-2a936530ad35/go.mod h1:tVouC34ewr4KABqqsVgsUmrg+eDeKsPIBD7VLOeZPEQ=
+github.com/codeready-toolchain/api v0.0.0-20210413031854-81652586fd90 h1:CN5cysQBZJl44K+dlFTzWsJxKqUDDco1li8ZS7QpNao=
+github.com/codeready-toolchain/api v0.0.0-20210413031854-81652586fd90/go.mod h1:tVouC34ewr4KABqqsVgsUmrg+eDeKsPIBD7VLOeZPEQ=
```

Comment:
please, run `go mod tidy` to clean the old items

Reply:
done